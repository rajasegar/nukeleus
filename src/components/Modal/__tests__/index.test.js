import React from 'react';
import { renderWithTheme } from '../../../test-utils';
import { Modal } from '../index';

describe('Modal component test suite', () => {
  const modalDiv = document.createElement('div');
  modalDiv.setAttribute('id', 'modal-root');
  it('should render Modal', async () => {
    const { container } = renderWithTheme(
      <Modal modalRootId="modal-root" overlay={true}>
        Modal Content
      </Modal>,
      {
        container: document.body.appendChild(modalDiv),
      }
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
