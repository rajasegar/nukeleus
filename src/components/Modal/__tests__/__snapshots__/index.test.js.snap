// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal component test suite should render Modal 1`] = `null`;
